crds/virtualmachineclasses.yaml

```
@@ -0,0 +1,440 @@
+          spec:
+            properties:
+              cpu:
+                description: CPU defines the requirements for the virtual CPU model.
+                properties:
+                  discovery:
+                    description: Create CPU model based on an intersection CPU features
+                      for selected nodes.
+                    properties:
+                      matchExpressions:
+                        description: matchExpressions is a list of label selector
+                          requirements. The requirements are ANDed.
+                        items:
+                          description: |-
+                            A label selector requirement is a selector that contains values, a key, and an operator that
+                            relates the key and values.
+                          properties:
+                            key:
+                              description: key is the label key that the selector
+                                applies to.
+                              type: string
+                            operator:
+                              description: |-
+                                operator represents a key's relationship to a set of values.
+                                Valid operators are In, NotIn, Exists and DoesNotExist.
+                              type: string
+                            values:
+                              description: |-
+                                values is an array of string values. If the operator is In or NotIn,
+                                the values array must be non-empty. If the operator is Exists or DoesNotExist,
+                                the values array must be empty. This array is replaced during a strategic
+                                merge patch.
+                              items:
+                                type: string
+                              type: array
+                          required:
+                          - key
+                          - operator
+                          type: object
+                        type: array
+                      matchLabels:
+                        additionalProperties:
+                          type: string
+                        description: |-
+                          matchLabels is a map of {key,value} pairs. A single {key,value} in the matchLabels
+                          map is equivalent to an element of matchExpressions, whose key field is "key", the
+                          operator is "In", and the values array contains only "value". The requirements are ANDed.
+                        type: object
+                    type: object
+                    x-kubernetes-map-type: atomic
+                  features:
+                    description: Required instructions for the CPU as a list More
+                      information about features [here](https://libvirt.org/formatdomain.html#cpu-model-and-topology)
```

"required" is ambiguous here.

```suggestion
                    description: A list of CPU instructions (features) required for virtual machines of this class.
                      More information about features [here](https://libvirt.org/formatdomain.html#cpu-model-and-topology)
```